### DISCUSSION TITLE
Text Scaling option?

### DISCUSSION THREAD

**Discussion by no-mood** [nix-community/stylix#462] [Q&A]
Hi everyone,
Could you confirm whether there's an option for text scaling (not fraction scaling)? I think this feature could be quite useful, especially since it's already possible to set font sizes via Stylix.

Additionally, having a text scaling option would be beneficial for abstraction purposes, considering that text scaling currently has to be manually configured on GTK and QT, as well as in each individual application that doesn't use these libraries.

Thank you!

**Comment by danth** [comment]
This doesn't exist currently.

**Reply by no-mood** [reply]
Would you be interested in this feature? I could work on it in a few weeks
